dhis-2/dhis-api/src/main/java/org/hisp/dhis/category/CategoryOption.java

```diff
@@ -56,6 +58,7 @@
     extends BaseDimensionalItemObject implements SystemDefaultMetadataObject
 {
     public static final String DEFAULT_NAME = "default";
+    public static final String EXTENDED_COMPARISON = "EXTENDED_COMPARISON";

```

Comment:
The name is a bit misleading, I Think.
Since, the feature is about allowing aggregation of NULL values what about `AGGREGATE_NULL_VALUES` or `NULL_VALUES_AGGREGATION` @larshelge may have a better idea